src/main/java/org/folio/circulation/domain/validation/LoanPolicyValidator.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain.validation;
+
+import static org.folio.circulation.support.results.Result.ofAsync;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Function;
+
+import org.folio.circulation.domain.LoanAndRelatedRecords;
+import org.folio.circulation.domain.policy.LoanPolicy;
+import org.folio.circulation.domain.validation.overriding.OverrideValidation;
+import org.folio.circulation.support.ValidationErrorFailure;
+import org.folio.circulation.support.results.Result;
+
+public class LoanPolicyValidator implements OverrideValidation {
+  private final Function<LoanPolicy, ValidationErrorFailure> itemLimitErrorFunction;
+
+  public LoanPolicyValidator(Function<LoanPolicy, ValidationErrorFailure> itemLimitErrorFunction) {
+    this.itemLimitErrorFunction = itemLimitErrorFunction;
+  }
+
+  @Override
+  public CompletableFuture<Result<LoanAndRelatedRecords>> validate(
+    LoanAndRelatedRecords records) {
+
```

Comment:
I did in this way because all implementation have method validate and it's not quite clear what exactly it validates. Calling private method refuseWhenItemIsNotLoanable() allows to see what exact validation will be done and understand the scenario why loan should be refused in particular validator.